### ISSUE TITLE
Accessibility issue: Scrollable region must have keyboard access on /notebook-ci-testing/notebooks/MIRI/Imaging/JWPipeNB-MIRI-imaging.html

### ISSUE THREAD

**Issue by dulude** [spacetelescope/notebook-ci-testing#109]
## What
An accessibility scan flagged the element `<div class="cell_output docutils container">` because scrollable region must have keyboard access. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/scrollable-region-focusable?application=playwright.

To fix this, ensure elements that have scrollable content are accessible by keyboard in safari.

Specifically:

Fix any of the following:
-   Element should have focusable content
-   Element should be focusable

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.1 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
